README.md

```diff
@@ -64,6 +64,37 @@ The build process also creates ```.stamps``` and ```.tmp-output```. The first co
 stamp files to keep track of which parts have been built, while the second contains
 intermediate build output (e.g., kernel images and ISO files).
 
+# Building Windows images
+
+You need to first get the ISO file for the Windows version you would like to install.
+You can download these images from MSDN. The hash and the name of the ISO is
+specified in the  ``images.json`` file. Place these files in the ``iso`` folder.
+
+S2E officially supports images listed in ``images.json``. Other images may
+work too but we did not test them. If you want to add support for new images,
+you will need to also update ``s2e.sys`` in the ``guest-tools`` repository in order
+to support the different kernels. Please refer to the documentation in ``guest-tools``.
```

Comment:
Can you link to guest-tool, i.e. [guest-tools](https://github.com/s2e/guest-tools)